contracts/examples/SimpleBlog.sol

```diff
@@ -2,16 +2,44 @@
 pragma solidity ^0.8.0;
 
 import "./SimpleFlatDirectory.sol";
+import "./SimpleComment.sol";
 
 contract SimpleBlog is SimpleFlatDirectory {
+    // event for EVM logging
+    event OwnerSet(address indexed oldOwner, address indexed newOwner);
+
+    // modifier to check if caller is owner
+    modifier isOwner() {
+        // If the first argument of 'require' evaluates to 'false', execution terminates and all
+        // changes to the state and to Ether balances are reverted.
+        // This used to consume all gas in old EVM versions, but not anymore.
+        // It is often a good idea to use 'require' to check if functions are called correctly.
+        // As a second argument, you can also provide an explanation about what went wrong.
+        require(msg.sender == owner, "Caller is not owner");
+        _;
+    }
+
     struct Blog {
         bytes title;
         uint256 timestamp; // 0 means deleted
     }
 
+    struct Comment {
```

Comment:
author and timestamp of the comment can also be stored in the directory with the content